alpha/apps/kaltura/lib/storage/urlTokenizers/kKalturaUrlTokenizer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+class kKalturaUrlTokenizer extends kUrlTokenizer
+{
+
+	/**
+	 * @var string
+	 */
+	protected $secret;
```

Comment:
The common term in tokenizer is key , lets use it + the base class gives us the getters